src/parse.jl

```diff
@@ -1,6 +1,9 @@
 using Dates: DateFormat, DatePart, min_width, max_width
 
 function tryparsenext_fixedtz(str, i, len, min_width::Int=1, max_width::Int=0)
+
+    str[i] == 'Z' && return ("Z", i+len)
```

Comment:
I believe this may interfere when parsing `Zulu`? We should probably verify this is the last character in the string